docs/admin/authentication.md

```diff
@@ -40,6 +40,29 @@ option to apiserver. The referenced file must contain one or more certificates a
 to use to validate client certificates presented to the apiserver. If a client certificate
 is presented and verified, the common name of the subject is used as the user name for the
 request.
+When using client certificate authentication, you should generate certificates by deployment
+script or yourself. The deployment script is implemented at
+`cluster/saltbase/salt/generate-cert/make-ca-cert.sh`
+Execute this script with two parameters. First is the `cert_ip` , second is the
+`subject-alt-name`. `cert_ip` used to generate the CA default CN and append to
+`subject-alt-name` for subjectAltName . http://wiki.cacert.org/FAQ/subjectAltName
+This script will make a ca.crt, a server.cert and a server.key. Finally, fill these parameters
+`--client-ca-file=/srv/kubernetes/ca.crt`
+`--tls-cert-file=/srv/kubernetes/server.cert`
+`--tls-private-key-file=/srv/kubernetes/server.key`
+If you want to make CA by yourself, these scripts you may need.
```

Comment:
I think it would be better to make the steps to generate a cert by yourself a different section (maybe in an appendix at the end) so that you can label them with numbers and provide more explanation of each step. Doing it inline in this paragraph is pretty confusing, and hopefully most folks don't need to do this by hand.